app/celery/provider_tasks.py

```diff
@@ -42,10 +42,7 @@ def deliver_throttled_sms(self, notification_id):
 
 # Celery rate limits are per worker instance and not a global rate limit.
 # https://docs.celeryproject.org/en/stable/userguide/tasks.html#Task.rate_limit
-# This task is dispatched through the `send-sms-tasks` queue.
-# This queue is consumed by 6 Celery instances with 4 workers in production.
-# The maximum throughput is therefore 6 instances * 4 workers = 24 tasks per second
-# if we set rate_limit="1/s" on the Celery task
```

Comment:
The infrastructure config changes but it's good to explain these limitations conceptually for folks who want to tweak the throughput. Maybe we could leave a documentation link that explains how that works?

Reply:
Added some docs explaining how the limit works.